### PR TITLE
[webui] application.js: drop unused functions

### PR DESCRIPTION
**Pull request by avindra**
These functions are not used anywhere, just dropping them.

### CODE REVIEW

File: src/api/app/assets/javascripts/webui/application.js [hunk 1]
```diff
@@ -37,46 +37,6 @@
...
-        document.forms[0].reset();
-        document.forms[0].elements[0].blur();
-        return;
-    }
-    window.open(url, 'helpwindow', 'toolbar=yes,location=yes,scrollbars=yes');
-    document.forms[0].reset();
-    document.forms[0].elements[0].blur();
-}
-
 function hide_dialog() {
```
Comment (mdeniz) [inline src/api/app/assets/javascripts/webui/application.js]: I guess that hide_dialog is also unused, isn't it?

Comment (avindra) [inline src/api/app/assets/javascripts/webui/application.js]: @mdeniz : Sure thing, will update the PR when I have a moment. Thanks!

**Comment by avindra** [comment]
Updated PR to drop `hide_dialog` as well.

**Comment by bgeuken** [comment]
Thanks for the patch

**Pull request merged**